engine/src/main/java/org/terasology/logic/characters/ServerCharacterPredictionSystem.java

```diff
@@ -214,11 +218,15 @@ public void update(float delta) {
             if (entry.getKey().equals(localPlayer.getCharacterEntity())) {
                 continue;
             }
-
-            setToTime(renderTime, entry.getKey(), entry.getValue());
```

Comment:
the "setToTime" method threw a null pointer exception in the "CharacterMovementSystemUtility" class. So I just skipped the object that was being deleted.